### ISSUE TITLE
[Accessibility Summary Authoring Guidelines] Abstract

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#152]
Now the abstract says:

> The Accessibility Summary's name is somewhat misleading. It is not intended to be a comprehensive  summary, but...

Perhaps starting with saying that the name is misleading is not starting off on the best of starts 😀. We could make explicit the purpose of the guide and metadata immediately, then tell the story by saying that for EPUB 1.0 it was mandatory, that it was meant to represent all accessibility information, that then with EPUB 1.1 it became optional and is used to explain what you cannot express with machine readable metadata.

**Comment by gregoriopellegrino** [comment]
I think we can close this
